src/firefox/index.js

```diff
@@ -89,12 +99,54 @@ export function createProfile(
 }
 
 
+/*
+ * Copies an existing Firefox profile and creates a new temporary profile.
+ * It resolves with the new profile object.
+ *
+ * The temporary profile will be deleted when the system process exits.
+ *
+ * The existing profile can be specified as a directory path or a name of
+ * one that exists in the current user's Firefox directory.
+ */
+export function copyProfile(
+    profileDirectory: string,
+    {copyFromUserProfile=FirefoxProfile.copyFromUserProfile}: Object = {})
```

Comment:
I suppose that this `copyFromUserProfile` option and its default value are mainly used to be able to inject a fake ```copyFromUserProfile``` method to simplify the testing of this helper.

how about using the same convention used in the other methods like this, by defining it in the module as a `const defaultCopyFromUserProfile`?

```
const defaultCopyFromUserProfile = FirefoxProfile.copyFromUserProfile;
...
export function copyProfile(
    profileDirectory: string,
    {copyFromUserProfile=defaultCopyFromUserProfile}: Object = {})
...
```